src/cluster_legacy.c

```diff
@@ -5856,6 +5868,11 @@ clusterNode *clusterNodeGetSlaveof(clusterNode *node) {
     return node->slaveof;
 }
 
+clusterNode *clusterNodeGetSlaveofMaster(clusterNode *node) {
```

Comment:
I think `clusterNodeGetMaster` is easier to grasp.

```suggestion
clusterNode *clusterNodeGetMaster(clusterNode *node) {
```

Reply:
yean, i updated it. Originally i was followed the previous function

```
clusterNode *clusterNodeGetSlaveof(clusterNode *node) {
    return node->slaveof;
}
```